app/components/modals/AboutModal.jsx

```diff
@@ -0,0 +1,94 @@
+import Modal from "./Modal";
+import { shell } from "electron";
+import { FormattedMessage as T } from "react-intl";
+import style from "./Modals.module.css";
+
+const AboutModal = ({ show, onCancelModal, version, updateAvailable }) => {
+  return (
+    <Modal className={style.about} {...{ show, onCancelModal }}>
+      <div className={style.aboutIcon} />
+      <div className={style.aboutContent}>
+        <div className={style.aboutTitle}>
+          <T id="aboutModal.decrediton" m="Decrediton" />
+        </div>
+        <div
+          className={style.infoCloseButtonTop}
+          onClick={onCancelModal}
+        />
+        <div className={style.aboutTextParagraph}>
+          <T
+            id="aboutModal.paragraph1"
+            m="A cross platform GUI Wallet for Decred written in node.js using Electron"
+          />
+        </div>
+        <div className={style.aboutTextParagraph}>
+          <T
+            id="aboutModal.paragraph2a"
+            m="Decrediton is free and open source software, developed and designed by the global team of"
+          />{" "}
+          <a
+            onClick={() =>
+              shell.openExternal("https://decred.org/contributors/")
```

Comment:
Given this wasn't a pure file move, I think you should go ahead and replace these inline links for the `ExternalLink` component

Reply:
Replaced.